src/firefly/js/core/background/JobHistory.jsx

```diff
@@ -0,0 +1,358 @@
+import {getBackgroundInfo, getJobInfo, isActive, isArchived, isDone, isFail, isSearchJob, isSuccess, Phase} from './BackgroundUtil';
+import {TablePanel} from '../../tables/ui/TablePanel';
+import {getAppOptions} from '../AppDataCntlr';
+import {dispatchBgJobInfo, dispatchBgSetInfo, dispatchJobCancel, dispatchJobRemove} from './BackgroundCntlr';
+import {InputField} from '../../ui/InputField';
+import Validate from '../../util/Validate';
+import {getRequestFromJob} from '../../tables/TableRequestUtil';
+import {dispatchTableAddLocal, dispatchTableSearch} from '../../tables/TablesCntlr';
+import {showInfoPopup} from '../../ui/PopupUtil';
+import {updateSet} from '../../util/WebUtil';
+import {download} from '../../util/fetch';
+import {showJobInfo} from './JobInfo';
+import {dispatchHideDropDown, dispatchShowDropDown} from '../LayoutCntlr';
+import {getTblById, processRequest} from '../../tables/TableUtil';
+import {SORT_DESC, SortInfo} from '../../tables/SortInfo';
+
+import InsightsIcon from '@mui/icons-material/Insights';
+import DownloadIcon from '@mui/icons-material/Download';
+import ReadMoreIcon from '@mui/icons-material/ReadMore';
+import StopCircleOutlinedIcon from '@mui/icons-material/StopCircleOutlined';
+import DeleteOutlineOutlinedIcon from '@mui/icons-material/DeleteOutlineOutlined';
+import {workingIndicator} from '../../ui/Menu';
+
+export function JobHistory({initArgs, help_id, width:pwidth, slotProps, ...props}) {
+    const {title, table, width:owidth, ...rest} = getAppOptions()?.background?.history?.props || {};
+    const width = owidth || pwidth || '65.5em';
+    const titleProps = {...slotProps?.title, ...title};
+    const tableProps = {...slotProps?.table, ...table};
+    return(
+        <Slot component={Stack} spacing={1} mt={1} sx={{flexGrow:1}} alignItems='center' slotProps={{...props, ...rest}}>
+            <Slot component={TitleSection}    sx={{width}} slotProps={titleProps}/>
+            <Slot component={JobHistoryTable} sx={{width}} slotProps={tableProps} help_id={help_id}/>
+        </Slot>
+    );
+}
+
+JobHistory.propTypes = {
+    initArgs: object,
+    help_id: string,
+    slotProps: shape({
+        title: Stack.propTypes,
+        table: Stack.propTypes
+    })
+};
+
+
+export function showBackgroundMonitor(show=true) {
+    if (show) {
+        dispatchShowDropDown({view:'BackgroundMonitorCmd', });
+    } else {
+        dispatchHideDropDown();
+    }
+}
+
+export function makeBackgroundMonitorMenuItem() {
+    const label = getAppOptions()?.background?.history?.label || 'Job History';
+    const TabRenderer =  React.forwardRef((props, ref) => {
+        const {jobs={}} = useStoreConnector(() => getBackgroundInfo());
+        const loading = Object.values(jobs).some((j) => isActive(j));
+        return (
+            <Tab {...props}>
+                {loading && <ListItemDecorator>{workingIndicator}</ListItemDecorator>}
+                {label}
+            </Tab>
+        );
```

Comment:
Add `ref` to the tab props.

I don't quite understand `forwardRef` (I think react is removing it in react 19). The `ref` object appears to be populated but the `setElement` function is not being called. It works if you add it to the props.
```
 <Tab {...{...props, ref}}>
                {loading && <ListItemDecorator>{workingIndicator}</ListItemDecorator>}
                {label}
 </Tab>
```